CanvasAPI/CanvasAPI.py

```diff
@@ -151,3 +152,97 @@ def getCoursesAssignmentsObjects(self, courseID):
                 .jsonObjects(object_hook=self.jsonObjectHook)
 
         return coursesAssignments
+
+    def getCoursesUsers(self, courseID, enrollmentType=None, **kwargs):
+        """
+        Get Canvas Users objects as requests Response object.  May be one of multiple pages.
+
+        :param courseID: ID number of the Canvas course object's User Objects to be retrieved
+        :type courseID: int
+        :param enrollmentType:
+        :type enrollmentType: str
+        :return:
+        :rtype: requests.models.Response
+        """
+        assert type(courseID) is int
+
+        if enrollmentType:
+            if not isinstance(enrollmentType, str):
+                raise TypeError('enrollmentType must be string')
+            if 'enrollment_type' not in kwargs:
+                kwargs['enrollment_type'] = str(enrollmentType)
+
+        queryURI = self._QueryURIs.COURSES_USERS.format(courseID=courseID)
+        response = self.get(queryURI, params=kwargs)
```

Comment:
`kwargs` values are assigned by any `key=value` arguments passed to any method if `key` does not match any declared argument for the method.  For example:

```
$ python
Python 2.7.10 (default, Aug 22 2015, 20:33:39)
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> def kwargtest(**kwargs):
...   import pprint
...   pprint.pprint(kwargs)
...
>>> kwargtest()
{}
>>> kwargtest('asdf')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: kwargtest() takes exactly 0 arguments (1 given)
>>> kwargtest(fubar='asdf')
{'fubar': 'asdf'}
```

This method will pass any keyword arguments along to `requests`.  There are many possible keywords that could be passed along, but I thought it was important to draw attention to `enrollment_type`, so I declared an argument for that parameter.  However, if that argument *AND* the keyword are both given, the keyword overrides the argument.